### PR TITLE
Fix doc

### PR DESCRIPTION
**Pull request by jburel**
fix intro text

cc @pwalczysko

### CODE REVIEW

File: docs/iviewer.rst [hunk 1]
```diff
@@ -1,9 +1,12 @@
 OMERO.iviewer
 =============
 
-In this document, we introduce two viewers in OMERO.web, the
-OMERO.iviewer (the main image viewer in OMERO, a 2D viewer which can
-open and browse multi-t, multi-z and multi-channel images).
+We introduce 
+OMERO.iviewer a 2D viewer which can
```
Comment (mtbc) [inline docs/iviewer.rst]:
```suggestion
OMERO.iviewer, a 2D viewer which can
```

File: docs/iviewer.rst [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+open and browse multi-t, multi-z and multi-channel images and allow to 
+draw and edit Regions of Interest.
+It also offers the ability to view several images and the same time and synchronize the
```
Comment (mtbc) [inline docs/iviewer.rst]:
```suggestion
It also offers the ability to view several images at the same time and synchronize the
```

File: docs/iviewer.rst [hunk 3]
```diff
@@ -1,9 +1,12 @@
 OMERO.iviewer
 =============
 
-In this document, we introduce two viewers in OMERO.web, the
-OMERO.iviewer (the main image viewer in OMERO, a 2D viewer which can
-open and browse multi-t, multi-z and multi-channel images).
+We introduce 
+OMERO.iviewer, a 2D viewer which can
+open and browse multi-t, multi-z and multi-channel images and allow to
```
Comment (pwalczysko) [inline docs/iviewer.rst]:
```suggestion
open and browse multi-t, multi-z and multi-channel images and allows to 
```

**Comment by pwalczysko** [comment]
On line https://github.com/ome/omero-guide-iviewer/pull/1/files#diff-82effecfde2e987acc2a86efce26ca3eR16

Replace
``How to open multidimensional images OMERO.iviewer`` with ``How to open multidimensional images in OMERO.iviewer`` (missing "in")

After that, ready to merge fmpov

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**